apps/web/components/Forms/ProposalForm.tsx

```diff
@@ -264,9 +239,37 @@ export const ProposalForm = ({
     chainId,
   });
 
+  const spendingLimitString = formatTokenAmount(
+    spendingLimit,
+    poolToken?.decimals ?? 0,
```

Comment:
```suggestion
    poolToken?.decimals ?? 18,
```

Like youd id for 
![image](https://github.com/user-attachments/assets/e1e6b1bc-3e8e-47ca-a956-458e346ec945)